public/controllers/management/components/management/ruleset/actions-buttons.js

```diff
@@ -86,25 +107,22 @@ class WzRulesetActionButtons extends Component {
             index: idx,
             uploaded: true,
             file: file,
-            error: 0
+            error: 0,
           });
         } catch (error) {
           errors = true;
           results.push({
             index: idx,
             uploaded: false,
             file: file,
-            error: error
+            error: error,
           });
         }
       }
       if (errors) throw results;
-      //ErrorHandler.info('Upload successful');
       return;
     } catch (error) {
-      if (Array.isArray(error) && error.length) return Promise.reject(error);
-      //TODO handle the erros
-      //ErrorHandler.handle('Files cannot be uploaded');
+      throw error;
     }
```

Comment:
**suggestion**: If we don't need to do anything in the `catch`, we could remove the `try/catch` block. If something in the function fails, the error will be propagated. But it is ok, anyways.

Reply:
Thanks for your advice @Desvelao